docs/openlaw-object/README.md

```diff
@@ -0,0 +1,2141 @@
+Openlaw.getSections(document.executionResult);
+```
+
+**Response**
+
+Returns an array of header names of [groupings](/markup-language/#groupings) in a template as strings.
+
+Example
+
+```js
+[
+  "Effective Date",
+  "Company Information",
+  "Employee Information",
+  "Other"
+]
+```
+
+### getVariableSections
+
+List a template's variables and conditionals that are organized under header names of [groupings](/markup-language/#groupings) in a template.
+
+```scala
+getVariableSections(
+  document: TemplateExecutionResult
+): js.Dictionary[js.Array[String]]
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `document` | [`TemplateExecutionResult`](#templateexecutionresult) | **Required.** The nested object returned from the [`execute` method](#execute) and [`executeForReview` method](#executeforreview). |
+
+Example
+
+```js
+// see examples above for #execute and #executeForReview for parameters
+const document = Openlaw.execute(compiledTemplate.compiledTemplate, {}, params);
+Openlaw.getVariableSections(document.executionResult);
+```
+
+**Response**
+
+Returns an object containing the header names of [groupings](/markup-language/#groupings) in a template and arrays of variable names associated with each grouping as strings.
+
+Example
+
+```js
+{
+  Company Information: [
+    "Company Name",
+    "Company Address",
+    "Company Signatory First Name",
+    "Company Signatory Last Name",
+    "Company Signatory Position"
+  ],
+  Effective Date: ["Effective Date"],
+  Employee Information: [
+    "Employee First Name",
+    "Employee Last Name",
+    "Employee Address",
+    "Employee Position",
+    "Position of Supervisor",
+    "Payment Start Date",
+    "Payment End Date",
+    "Salary in Ether",
+    "Recipient Address",
+    "Employee Responsibilities",
+    "Days of Vacation"
+  ],
+  Other: [
+    "Additional Agreements",
+    "Confidentiality Agreement",
+    "Dispute Resolution",
+    "Governing Law"
+  ]
+}
+```
+
+### isDeal
+
+Check if template is a [deal](/markup-language/#deals) template.
+
+```scala
+isDeal(
+  template: CompiledTemplate
+): Boolean
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `template` | [`CompiledTemplate`](#compiledtemplate) | **Required.** The nested object returned from the [`compileTemplate` method](#compiletemplate). |
+
+Example
+
+```js
+const compiledTemplate = Openlaw.compileTemplate('<%\n==Effective Date==\n[[Effective Date: Date]]\n\n==Company Name and Address==\n[[Company Name]]\n[[Company Address:Address]]\n[[Corporation:YesNo \"Is the company a corporation?\"]]\n[[LLC:YesNo \"An LLC?\"]]\n...[[Employee Offer Letter: Template(\"Employee Offer Letter\")]]...');
+Openlaw.isDeal(compiledTemplate.compiledTemplate);
+```
+
+**Response**
+
+Returns `true` if template is a [deal](/markup-language/#deals) template.
+
+## Variable
+
+### showInForm
+
+Check if variable is shown in the input form in draft mode.
```

Comment:
Let's explain a bit more what this means.

Reply:
I updated the language here with a better explanation. @outkaj and @adridadou can you check if that's sufficient?